cmd/juju/commands/bootstrap.go

```diff
@@ -961,10 +961,13 @@ See `[1:] + "`juju kill-controller`" + `.`)
 		}
 	}
 
-	if cloud.Type == k8sconstants.CAASProviderType &&
-		cloud.HostCloudRegion == caas.K8sCloudOther &&
-		bootstrapParams.ControllerServiceType == "" {
-		logger.Warningf("bootstrapping to an unknown kubernetes cluster should be used with option --config controller-service-type. See juju help bootstrap")
+	if cloud.Type == k8sconstants.CAASProviderType {
+		k8sType := string(cloud.HostCloudRegion)
+		if cloud.HostCloudRegion == caas.K8sCloudOther {
+			k8sType = "unknown"
```

Comment:
Can we just put a log message in the unknown case to be
ctx.Infof("Unable to identity bootstrapped Kubernetes cluster")

Reply:
yes we can